docs/src/tutorial/annotation_typing_table.csv

```diff
@@ -0,0 +1,9 @@
+Feature             ,Cython 0.29        ,Cython 3.0                
+``int``,Any Python object,Exact Python ``int``
```

Comment:
For now I'm unclear whether this is a desired change (see https://github.com/cython/cython/issues/4944)